README.md

```diff
@@ -3,13 +3,13 @@
 [![npm version](https://img.shields.io/npm/v/strapi-provider-upload-cloudflare.svg)](https://www.npmjs.org/package/strapi-provider-upload-google-cloud-storage)
 [![npm downloads](https://img.shields.io/npm/dm/strapi-provider-upload-cloudflare.svg)](https://www.npmjs.org/package/strapi-provider-upload-google-cloud-storage)
 
-**Non-Official** Cloudflare Upload Provider for Strapi
+**Non-Official** Cloudflare Images and Video Upload Provider for Strapi
 
 This has only been tested with Strapi 4.
 
 To find your Cloudflare accountId and apiKey, log into Cloudflare and click "Images". On that page you should see "Account ID" under Developer Resources. For apiKey, if you click the "Use API" tab you'll see a link next to "API Token" to generate an apiKey. The only permission that API Token needs is "Account.Cloudflare Images".
 
-If you have multiple image variants the first one will be selected. If you'd like to be able to pick a specific image variant, make an issue or pull-request. :) 
+If you have multiple image variants the first one will be selected. If you'd like to be able to pick a specific image variant, make an issue or pull-request. :)
```

Comment:
Could we fix this piece of documentation?

It would actually be great if the selected variant for the cms ('cms' in our case) was set using a configuration parameter somewhow

Reply:
I have added 'variant' to the config parameters, same as the two IDs so hopefully that'll work properly and allow us to have a degree of flexibility. I also modified the documentation accordingly.